english/programmer/lead_programmer.md

```diff
@@ -0,0 +1,8 @@
+You raise your hand slowly, but just as you did you notice in your peripheral vision that someone else has raised his hand.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.

Reply:
Please review this, @casey-collab